cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -632,6 +633,7 @@ def _get_volume_stats(self):
                         'pool_state': pool_state,
                         'pool_down_reason': pool_down_reason,
                         'custom_attributes': custom_attributes,
+                        'independent_snapshots': independent_snapshot,
```

Comment:
This has no effect, right? Is it a way to try to report if the pool supports independent snapshots? Because it doesn't, it only reports the setting being set - which is only honored by the scheduler, which only looks at the volume-type.

I don't think this is the right way to report a setting being active. Please consider adding a custom, admin-only API endpoint for stuff like this if you need it reported via API.

It also has the problem, that cinder-scheduler and cinder-volume use independent configs as they're independent services, so this one might return "True" while cinder-scheduler has it set "False".

Reply:
It currently has no effect, but this is needed as the first step. 
The second step is to manually adde the extra spec in the volume type for the vmware volume type.  This will ensure that when a user wants a vmware volume type it will match and the scheduler will read this extra spec value.
This is exactly how it's supposed to work between the scheduler, the driver capabilities and the volume type extra specs.  This is exactly how capabilities are supposed to be reported by a driver.

Reply:
I probably missed and forgot the magic behind volume-type properties automatically matching the pools' properties.